### PR TITLE
Ui refresh tab setup

### LINKED ISSUE #1100: Implement tabbed navigation
**The product definition, design, content, and assets are a WIP and are neither finalized nor fully approved by WHO. The purpose of this task is to prototype the feature so we can execute quickly when the non-engineering deliverables are finalized.**

<details>
<summary>Rough mock (subject to change)</summary>
Screen Shot 2020-04-22 at 5 53 34 PM
</details>

### PR DESCRIPTION
**Pull request by theswerd**
### UI Refresh Tab Setup

Closes #1100 

#### Screenshots
iOS:
Simulator Screen Shot - iPhone 11 Pro Max - 2020-04-24 at 09 25 48
Android:
Screenshot_1587746004

What I did:
- I added back the Cupertino Icons package thats comes pre-packed into Flutter by default.
- I deleted the old homepage
- I created a new directory Main_pages for the central pages
- I created a new file called home_page_router.dart that serves as the tab bar where all the home pages will be present on
- I edited the main.dart file to make our app a Cupertino app

#### How did you test the change?
* [x] iOS Simulator
* [x] Android Emulator

---

#### Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).
- [x] REQUIRED: Do you have an Issue **assigned to you** for this PR? 
- [x] Provided detailed pull request description and a succinct title
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

### CODE REVIEW

**Comment by britannio** [comment]
We'll need to update these icons

File: client/flutter/lib/pages/main_pages/home_page_router.dart [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            return null;
+        }
+      },
+      tabBar: CupertinoTabBar(
+        inactiveColor: Colors.black,
+        activeColor: Constants.accent,
+
+        items: [
+          BottomNavigationBarItem(
+              icon: Icon(CupertinoIcons.home), title: Text("Home")),
```
Comment (hspinks) [inline client/flutter/lib/pages/main_pages/home_page_router.dart:46]: please mark a TODO & open an issue for localizing the text here

```suggestion
              // TODO: localize title strings
              icon: Icon(CupertinoIcons.home), title: Text("Home")),
```

File: client/flutter/lib/pages/main_pages/home_page_router.dart [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:who_app/constants.dart';
+import 'package:who_app/pages/main_pages/check_up_page.dart';
+import 'package:who_app/pages/main_pages/home_page.dart';
+import 'package:who_app/pages/main_pages/learn_page.dart';
+import 'package:who_app/pages/main_pages/settings_page.dart';
+import 'package:who_app/pages/main_pages/stats_page.dart';
+
+class HomePageRouter extends StatelessWidget {
```
Comment (hspinks) [inline client/flutter/lib/pages/main_pages/home_page_router.dart:11]: This is nitpicky and not a blocker for this PR, but we may want to consider naming this something that indicates it's the main tab-based router for the application as a whole, e.g. `MainTabRouter` / `AppTabRouter` or something along those lines.

The reason for this being: we may have a sub-router or navigation within the Home tab of the app, and that could get confusing to see multiple routers with Home in the name.

File: client/flutter/ios/Runner.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -331,7 +331,6 @@
 /* Begin XCBuildConfiguration section */
 		249021D3217E4FDB00AE95B9 /* Profile */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = 7AFA3C8E1D35360C0083082E /* Release.xcconfig */;
```
[the remark this answers is not in the document]
Comment (theswerd) [reply client/flutter/ios/Runner.xcodeproj/project.pbxproj:334]: I have no idea

**Pull request closed without merging**